frontend/src/scenes/funnels/FunnelBarGraph.tsx

```diff
@@ -525,7 +525,7 @@ export function FunnelBarGraph(props: ChartParams): JSX.Element {
                                             (
                                             {percentage(
                                                 step.order > 0 ? step.count / steps[stepIndex - 1].count : 1,
-                                                1,
+                                                2,
```

Comment:
Would probably be better to use `undefined` instead, since that'd defer to the default, which is in fact `2` – but that's minor

Reply:
We're explicitly doing 2 elsewhere in funnels though